lib/galaxy/jobs/runners/util/pykube_util.py

```diff
@@ -10,6 +10,7 @@
         Job,
         Pod
     )
+    from pykube.exceptions import HTTPError
```

Comment:
Need to add ``HTTPErrror = None`` in the ImportError handling below.

```
galaxy/jobs/runners/kubernetes.py:18: in <module>
    from galaxy.jobs.runners.util.pykube_util import (
E   ImportError: cannot import name 'HTTPError'
```

Reply:
I also added HTTPClient